resources/lib/ipwww_radio.py

```diff
@@ -178,7 +178,7 @@ def AddAvailableLiveStreamItem(name, channelname, iconimage):
             title = name + ' - [I][COLOR %s]%s Kbps[/COLOR] [COLOR white]%s[/COLOR] [COLOR grey]%s[/COLOR][/I]' % (
                 quality_colours[quality], quality_bitrates[quality] , location_names[location], provider_name)
 
-            PlayStream(title, url, iconimage, '', '')
+            PlayStream(name, url, iconimage, '', '')
```

Comment:
I just added all the info from the re match list. It was useful for debugging and the users can see what they are receiving because the bitrate doesn't show up in the OSD overlay. It is easier to have a conversation with users when they can see what stream they are receiving in the gui. 

I use the kbps numbers in video to compare with the m3u8 bitrate, so I was trying to keep the numbers consistent and synchronised.